scapy/config.py

```diff
@@ -9,6 +9,7 @@
 
 import os,time,socket,sys
 
+import scapy
```

Comment:
Could you just use `from scapy import VERSION` instead? Apart from this I like your patch!